packages/nextjs/app/builders/0x558d82B022a8A42E9684BaCaA97a35318d4C293A/page.tsx

```diff
@@ -0,0 +1,82 @@
+"use client";

```

Comment:
We don't need this restriction since your page doesn't use any client-only mechanism.

Please remove this line so we can improve the app's performance by pre-rendering the page during the compile process.